Error running docker-compose

Following the `docker-compose up` command included in the README gives me the following error  on OS X:

```
$ docker-compose up
Starting iot-center-v2_influxdb_v2_1 ... done
Starting iot-center-v2_iot_center_1  ... done
Attaching to iot-center-v2_influxdb_v2_1, iot-center-v2_iot_center_1
iot_center_1   | internal/modules/cjs/loader.js:818
iot_center_1   |   throw err;
iot_center_1   |   ^
iot_center_1   |
iot_center_1   | Error: Cannot find module '/app/run.sh'
iot_center_1   |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
iot_center_1   |     at Function.Module._load (internal/modules/cjs/loader.js:667:27)
iot_center_1   |     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
iot_center_1   |     at internal/main/run_main_module.js:17:47 {
iot_center_1   |   code: 'MODULE_NOT_FOUND',
iot_center_1   |   requireStack: []
iot_center_1   | }
iot-center-v2_iot_center_1 exited with code 1
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -1,6 +1,5 @@
 version: "3"
 services:
-
   influxdb_v2:
     image: quay.io/influxdb/influxdb:2.0.0-rc
     ports:
@@ -18,6 +17,6 @@ services:
     volumes:
       - $PWD/app:/app
     working_dir: /app
-    command: /app/run.sh
+    command: ./run.sh
     depends_on:
       - influxdb_v2

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -1,11 +1,11 @@
 version: "3"
 services:
   influxdb_v2:
-    image: quay.io/influxdb/influxdb:2.0.0-rc
+    image: quay.io/influxdb/influxdb:v2.0.2
     ports:
-      - "9999:9999"
+      - "8086:8086"
     environment:
-      - INFLUXD_HTTP_BIND_ADDRESS=:9999
+      - INFLUXD_HTTP_BIND_ADDRESS=:8086
     command: influxd --reporting-disabled
 
   iot_center:
@@ -13,7 +13,7 @@ services:
     ports:
       - "5000:5000"
     environment:
-      - INFLUX_URL=http://influxdb_v2:9999
+      - INFLUX_URL=http://influxdb_v2:8086
     volumes:
       - $PWD/app:/app
     working_dir: /app

